code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm

```diff
@@ -235,7 +235,7 @@ GLOBAL_LIST_INIT(wraith_banish_very_short_duration_list, typecacheof(list(
 
 	for(var/mob/living/living_contents in banishment_target.GetAllContents()) //Safety measure so living mobs inside the target don't get lost in Brazilspace forever
 		contained_living += living_contents
-		living_contents.apply_status_effect(/datum/status_effect/incapacitating/unconscious)
+		living_contents.apply_status_effect(STATUS_EFFECT_STUN)
```

Comment:
unrelated change

Reply:
It's needed since flying vehicles check the pilot's stat var while flying. If a Wraith banishes a flying vehicle, it forces it to crash. Stun is mechanically the same as unconscious and prevents this interaction.